src/components/SubscriptAvatar.js

```diff
@@ -47,45 +51,60 @@ function SubscriptAvatar(props) {
     if (props.noMargin) {
         marginStyle = {};
     }
+
+    const mainAvatar = (
+        <Avatar
+            containerStyles={StyleUtils.getWidthAndHeightStyle(StyleUtils.getAvatarSize(props.size || CONST.AVATAR_SIZE.DEFAULT))}
+            source={props.mainAvatar.source}
+            size={props.size || CONST.AVATAR_SIZE.DEFAULT}
+            name={props.mainAvatar.name}
+            type={props.mainAvatar.type}
+        />
+    );
+
+    const secondaryAvatar = (
+        <View
+            style={[props.size === CONST.AVATAR_SIZE.SMALL_NORMAL ? styles.flex1 : {}, isSmall ? styles.secondAvatarSubscriptCompact : subscriptStyle]}
+            // Hover on overflowed part of icon will not work on Electron if dragArea is true
+            // https://stackoverflow.com/questions/56338939/hover-in-css-is-not-working-with-electron
+            dataSet={{dragArea: false}}
+        >
+            <Avatar
+                iconAdditionalStyles={[
+                    StyleUtils.getAvatarBorderWidth(isSmall ? CONST.AVATAR_SIZE.SMALL_SUBSCRIPT : CONST.AVATAR_SIZE.SUBSCRIPT),
+                    StyleUtils.getBorderColorStyle(props.backgroundColor),
+                ]}
+                source={props.secondaryAvatar.source}
+                size={isSmall ? CONST.AVATAR_SIZE.SMALL_SUBSCRIPT : CONST.AVATAR_SIZE.SUBSCRIPT}
+                fill={themeColors.iconSuccessFill}
+                name={props.secondaryAvatar.name}
+                type={props.secondaryAvatar.type}
+            />
+        </View>
+    );
```

Comment:
Here I only put the components in variables so we can conditionally render the tooltip and avoid duplicating code